amr-wind/physics/multiphase/VolumeFractions_K.H

```diff
@@ -0,0 +1,231 @@
+#ifndef VolumeFraction_K_H_
+#define VolumeFraction_K_H_
+
+#include <AMReX_FArrayBox.H>
+#include <cmath>
+
```

Comment:
Should be in `amr_wind` namespace. Don't use `_K.H` in newly introduced files. We don't support 2D. Prefer reserving camelcase names for headers with class definitions.

Reply:
Done !